$NetBSD: patch-doc_Makefile.am,v 1.3 2020/03/11 10:54:43 wiz Exp $

Generate html docs in build phase (instead of install phase).
gnome-doc-tool seems to ignore "-o html" so manually generate
the same effect.

Solves PLIST problem due to missing html files.

This is not known to be reported upstream.

--- doc/Makefile.am.orig	2017-12-31 12:31:21.000000000 +0000
+++ doc/Makefile.am
@@ -7,7 +7,7 @@ EXTRA_DIST = docbook2html.sh docbook
 html/GuideIndex.html: docbook/GuideIndex.xml
 	rm -rf html; mkdir html; cp $(srcdir)/icons/* html/
 	if [ -x "$(GNOME_DOC_TOOL)" ]; then \
-		"$(GNOME_DOC_TOOL)" html -o html $(srcdir)/docbook/GuideIndex.xml ; \
+		cd html && "$(GNOME_DOC_TOOL)" html $(srcdir)/../docbook/GuideIndex.xml ; \
 	else \
 		echo "gnome-doc-tool not found, html is not built" ; \
 	fi
